.github/workflows/DetectCDKBootstrapVersionChanges.yml

```diff
@@ -1,49 +1,115 @@
 name: Detect CDK Bootstrap Version Changes
 
-on: [pull_request]
+on:
+  schedule:
+    # Runs at 00:00 UTC every Monday
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+  id-token: write
 
 jobs:
   detect-cdk-bootstrap-changes:
     runs-on: ubuntu-latest
     steps:
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: ${{ secrets.RELEASE_WORKFLOW_ACCESS_TOKEN_ROLE_ARN }}
+        aws-region: us-west-2
+
+    - name: Retrieve secret from AWS Secrets Manager
+      uses: aws-actions/aws-secretsmanager-get-secrets@v2
+      with:
+        secret-ids: |
+          AWS_SECRET, ${{ secrets.RELEASE_WORKFLOW_ACCESS_TOKEN_NAME }}
```

Comment:
reusing the same token that sync-dev main workflow uses because thats the easiest